### ISSUE TITLE
Beacon Chain Client: error "Failed to handle p2p pubsub" error="could not process attestation from fork choice service: pre state of target block 229896 does not exist"

### ISSUE THREAD

**Issue by WhisperingChaos** [prysmaticlabs/prysm#4295]
msg="Starting beacon node" prefix=node version="Git commit: a9a5973b983c8293148cc59403221cfab4112081. Built at: 2019-12-12 22:57:27+00:00"

Seems like duplicate of #3819, as even the timing is similar.  However, message emission doesn't always follow 15min interval.  

```
...
time="2019-12-13 14:47:04" level=info msg="Executing state transition on block" prefix=forkchoice root=0x590a70b0... slot=229895
time="2019-12-13 14:47:04" level=info msg="Finished applying state transition" attestations=4 deposits=0 prefix=blockchain slot=229895
time="2019-12-13 14:47:19" level=info msg="Executing state transition on block" prefix=forkchoice root=0xe0f350b7... slot=229896
time="2019-12-13 14:47:19" level=error msg="Failed to handle p2p pubsub" error="could not process attestation from fork choice service: pre state of target block 229896 does not exist" prefix=sync topic="/eth2/beacon_attestation/ssz"
time="2019-12-13 14:47:19" level=error msg="Failed to handle p2p pubsub" error="could not process attestation from fork choice service: pre state of target block 229896 does not exist" prefix=sync topic="/eth2/beacon_attestation/ssz"
**above message repeats until** 
time="2019-12-13 14:47:20" level=info msg="Starting next epoch" epoch=28737 finalizedEpoch=28735 justifiedEpoch=28736 prefix=forkchoice previousJustifiedEpoch=28735
time="2019-12-13 14:47:20" level=info msg="Validator registry information" activeValidators=577 averageBalance="4.77643 ETH" prefix=forkchoice totalValidators=874
time="2019-12-13 14:47:20" level=info msg="Finished applying state transition" attestations=7 deposits=0 prefix=blockchain slot=229896
```

**Comment by torfbolt** [comment]
Can confirm, same error here occasionally with latest beacon client version (30b4b045f503a52142a9f26a8d614b4d226b44eb).

Looking at the validator balance it seems like this prevents proper validation, as my client accrues negative rewards during the duration of this beacon client error.

**Comment by 0xKiwi** [comment]
Hello! Can anyone still reproduce this issue or can we consider it solved?

**Comment by torfbolt** [comment]
Just saw this again with the latest version (68c1ca755d7bd5ffeab337ccf85c138f7f89778e):
`pre state of slot 62821 does not exist`

**Comment by rauljordan** [comment]
Resolved by @terencechain in #4740

**Comment by torfbolt** [comment]
Still seeing this quite regularly in the latest docker images after #4740. It seems to happen often enough that my validator has a net negative reward.
